### DISCUSSION TITLE
A persistent file gets corrupted after windows error (Crash)

### DISCUSSION THREAD

**Discussion by Ant9cp** [Monika-After-Story/MonikaModDev#9151] [QA - tech support + general] [answered]
I've been having some issue's with my windows lately, everytime this blue windows error screen come's up, then after when my pc restarts i then look in the charactars folder i get a message saying a persistent file has been corrupted why does that happen? why don't i just get that scene with monika saying another crash (name)?

**Comment by multimokia** [comment, marked as answer]
Because the persistent wasn't saved properly and has invalid data, making it unloadable due to the OS crash. As such since it can't be loaded, we fall back to the next working available backup and load it

**Comment by Ant9cp** [comment]
But does that also happen before a persistent file getting corrupted in a older version of MAS?, when monika notices your pc or the game crashing? when she actually mention. Another crash (name)

**Comment by Booplicate** [comment]
When Monika mentions a crash, it means (most *likely*) nothing bad actually happened. When you get the note, it means your persistent got corrupted. Both are possible to get.
